dash-pipeline/SAI/saithrift/Makefile

```diff
@@ -9,62 +9,57 @@ META=../SAI/meta
 
 SAIRPC_VENDOR_EXTRA_LIBS?=
 SAIRPC_EXTRA_LIBS= "$(SAIRPC_VENDOR_EXTRA_LIBS) \
-		-L/lib/x86_64-linux-gnu -Wl,-rpath=/lib/x86_64-linux-gnu -lm \
+		-L/lib/x86_64-linux-gnu -Wl,-rpath=/lib/x86_64-linux-gnu \
 		-L/usr/local/lib/ -Wl,-rpath=/usr/local/lib \
-	    -lpthread \
-	    -lpiprotogrpc \
-	    -lpiprotobuf \
-	    -lprotobuf \
-	    -lgrpc++ \
-	    -lgrpc \
-	    -lpiall \
-	    -lpi_dummy \
-	    -lpthread \
-	    -labsl_synchronization \
-	    -labsl_status \
-		-labsl_raw_hash_set \
-		-lgpr \
-		-lre2 \
-		-lssl \
-		-laddress_sorting"
+	    -lprotobuf -lpiprotobuf -lpiprotogrpc -lgrpc++"
```

Comment:
I'm surprised it still works with all these extra libs trimmed out. Initially, I'd painstakingly added them in one at a time by trial and error, until build succeeded. I wonder why it builds w/o them all now?

Reply:
they are not needed, maybe there were needed at some point, but those 4 are required only now